src/App.js

```diff
@@ -1,12 +1,27 @@
-import { Routes, Route } from 'react-router-dom';
-import Splash from './pages/Splash';
+import React, { useState } from 'react';
+import { useLocation } from 'react-router-dom';
+import MenuOpener from './components/NavBar/MenuOpener';
+import Navbar from './components/NavBar/Navbar';
+import RouterConfig from './navigation/RouterConfig';
 
 function App() {
```

Comment:
- [ ] Please, kindly refactor this function and other functions in your project to ES6 arrow function, so your code complies with the latest standards.

The function keyword has been soft deprecated all over the software industry.  This means, almost nobody uses it anymore, so much that it's considered non best practice.




## Explanation

The `function` keyword has been soft deprecated in almost all the software industry in favor of ES6 arrow functions.




## Tips and extra help

You can do a search in Vscode for the "function" word, and correct as needed.  Shouldn't take you more than 5-10 min so don't worry.

![image](https://user-images.githubusercontent.com/57421823/205704596-603147a3-1f90-4f40-aed9-77df7d13cc88.png)

Refactoring is easy:

```javascript
const App = () => {
  // ...
}
```